examples/with-federated-passkeys/src/pages/api/subOrg.ts

```diff
@@ -0,0 +1,85 @@
+import type { NextApiRequest, NextApiResponse } from "next";
+import {
+  TurnkeyApi,
+  init as httpInit,
+  withAsyncPolling,
+  FederatedRequest,
+  TurnkeyApiTypes,
+} from "@turnkey/http";
+import axios from "axios";
+
+type TAttestation = TurnkeyApiTypes["v1Attestation"];
+
+type CreateSubOrgRequest = {
+  orgName: string;
+  challenge: string;
+  attestation: TAttestation;
+};
+
+type CreateSubOrgResponse = {
+  subOrgId: string;
+};
+
+type ErrorMessage = {
+  message: string;
+};
+
+httpInit({
+  apiPublicKey: process.env.API_PUBLIC_KEY!,
+  apiPrivateKey: process.env.API_PRIVATE_KEY!,
+  baseUrl: process.env.BASE_URL!,
+});
+
+export default async function createUser(
+  req: NextApiRequest,
+  res: NextApiResponse<CreateSubOrgResponse | ErrorMessage>
+) {
+  const createUserRequest = req.body as CreateSubOrgRequest;
+
+  const createUserMutation = withAsyncPolling({
+    request: TurnkeyApi.postCreateSubOrganization,
+  });
+
+  try {
+    const createUserActivity = await createUserMutation({
```

Comment:
All of these should be renamed to "createSubOrganization" instead of "createUser"